x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/FileSettingsRoleMappingsStartupIT.java

```diff
@@ -73,6 +74,10 @@ protected Path nodeConfigPath(int nodeOrdinal) {
         return null;
     }
 
+    public void setupLogging() {
+        updateClusterSettings(Settings.builder().put("logger.org.elasticsearch.common.file", "DEBUG"));
```

Comment:
An alternative way of enabling test logging you might consider is using the TestLogging annotation (on the test method):

```
@TestLogging(value = "org.elasticsearch.common.file:DEBUG", reason = "https://github.com/elastic/elasticsearch/issues/98391")
```

Reply:
good hint. changed to used @TestLogging